docs/reference/tctl/workflow.md

```diff
@@ -0,0 +1,33 @@
+---
+id: workflow
+title: tctl workflow
+description: How to operate Workflow Executions using tctl.
+tags:
+  - reference
+  - tctl
+---
+
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowExecution from '../../content/what-is-a-workflow-execution.md'
+
+The `tctl workflow` commands operate <preview page={WhatIsAWorkflowExecution}>Workflow Executions</preview>.
+
+## tctl workflow show
```

Comment:
What about this?

```suggestion
## `show`
```

Reply:
That would parallel the individual subcommand pages, wouldn't it? Done!